### DISCUSSION TITLE
Hide @document.meta

### DISCUSSION THREAD

**Discussion by kishikaisei** [nvim-neorg/neorg#149] [Q&A]
So as the title says, is there a way to hide the @document.meta, or keep the fold closed by default?
I don't really need it to be open all the time.

**Comment by vhyrro** [comment]
Unfortunately not that I know of. I would've implemented it a while back if that were the case :P. Currently treesitter folds only allow you to give nodes an `@fold` attribute, but I don't believe that you can apply a priority to said folds just yet. This would mean that if you want to fold TS nodes by default you'd end up folding *everything*, including all headings, lists and ranged tags like `@document.meta`, `@code` etc. :(
